### PR TITLE
API Updates

### PR DESCRIPTION
**Pull request by Armando-Ibarra**
Added API key auth to Integrated Platforms Balances (CS2)
Added support for Trustly & Cv Connect APM (CS2)
Review payment & payout request schema
Added patch endpoint for payment instruments (CS2)

### CODE REVIEW

File: src/CheckoutSdk/Payments/Sender/PaymentIndividualSender.cs [hunk 1]
```diff
@@ -9,15 +9,17 @@ public PaymentIndividualSender() : base(PaymentSenderType.Individual)
...
 
         public Address Address { get; set; }
 
         public AccountHolderIdentification AccountHolderIdentification { get; set; }
         
-        public string MiddleName { get; set; }
-
         public string ReferenceType { get; set; }
 
         public string DateOfBirth { get; set; }
```
Comment (armandojaleo) [inline src/CheckoutSdk/Payments/Sender/PaymentIndividualSender.cs:25]: DateOfBird is Dob, no? Probably it must be deleted

File: src/CheckoutSdk/Payments/ProcessingSettings.cs [hunk 2]
```diff
@@ -58,9 +59,14 @@ public class ProcessingSettings
...
 
         public IList<string> CustomPaymentMethodIds { get; set; }
+        
+        public string MerchantCallbackUrl { get; set; }
 
         public long? ShippingDelay { get; set; }
 
         public IList<ShippingInfo> ShippingInfo { get; set; }
+        
+        [JsonProperty(PropertyName = "senderInformation")]
```
Comment (armandojaleo) [inline src/CheckoutSdk/Payments/ProcessingSettings.cs:69]: sender or senderInformation?

File: src/CheckoutSdk/Payments/Sender/PaymentIndividualSender.cs (hunk 1 above)
Comment (Armando-Ibarra) [reply src/CheckoutSdk/Payments/Sender/PaymentIndividualSender.cs:25]: Each belongs to different version, `dob` for CS2 and `date_of_birth` for CS1, that's why we keep both

File: src/CheckoutSdk/Payments/ProcessingSettings.cs (hunk 2 above)
Comment (Armando-Ibarra) [reply src/CheckoutSdk/Payments/ProcessingSettings.cs:69]: `senderInformation`

**Review by armandojaleo — APPROVED** [review]

**Pull request merged**